apps/dashboard/app/(app)/apis/[apiId]/settings/default-prefix.tsx

```
@@ -50,44 +60,61 @@ export const DefaultPrefix: React.FC<Props> = ({ keyAuth }) => {
+        <Card>
+          <CardHeader>
+            <CardTitle>Default Prefix</CardTitle>
+            <CardDescription>
+              Set default prefix for the keys under this API. Don't add a trailing underscore, we'll
+              do that automatically
+            </CardDescription>
+          </CardHeader>
+          <CardContent>
+            <div className="flex flex-col space-y-2">
+              <input type="hidden" name="keyAuthId" value={keyAuth.id} />
+              <label className="hidden sr-only">Default Prefix</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)